Documentation/policy/language.rst

```diff
@@ -143,8 +143,9 @@ the label ``role=frontend``.
 Egress Allow All
 ~~~~~~~~~~~~~~~~~
 
-An empty `EndpointSelector` will select all endpoints, thus writing a rule that will allow
-all egress traffic from an endpoint may be done as follows:
+An empty `EndpointSelector` will select all endpoints in the pod's namespace,
```

Comment:
>in the pod's namespace

Well, technically, this would be the CNP namespace ("default" if not specified).

Reply:
Thanks, PR updated.